geoips/geoips_utils.py

```diff
@@ -90,6 +90,11 @@ def load_all_yaml_plugins():
 
             # Set some additional information on the YAML plugin
             # The name of the package the plugin comes from
+            if yaml_plugin is None:
```

Comment:
Should this be if yaml_plugin rather than is None?

Reply:
I updated this. It's correct to test that it is false so it is now `if not yaml_plugin:`.